exercises/exercise3.js

```diff
@@ -1,12 +1,56 @@
 // Set this assignment to true if you do want to use it.
-module.exports.ACTIVATE_BOT = false;
+module.exports.ACTIVATE_BOT = true;
 
 module.exports.botScripts = [
   {
-    label: 'An Example Script',
-    prompt: 'Good Evening',
+    label: 'Would you rather!',
+    prompt: 'Would you rather go shopping or out to eat?',
     handler: function () {
-      return 'Good Evening to You!';
+      return 'its 2020, solid food is for the weak! lets go shopping';
     },
+    isCaseSensitive: false,
+    isReply: true,
+    isLitening: true
   },
+
+  {
+    label: 'Would you rather the aliens that make first contact be organics or robots',
+     prompt: 'I would rather #{name} make first contact. ',
+      handler: function (params) {
+        
+        if ( params.name === "robots"  ){
+          return " really?" + params.name1 + "would be my last choice"
+        }
+        else  {
+          return "See your smart!";
+        }
+      },
+      isCaseSensitive: false,
+      isReply: true,
+      isLitening: true
+    },
+      {
+        label: "Dance break",
+        prompt:"#{playerTwo} challanges #{playerOne} to a dance off, who will win?",
+        handler: function (params) {
+          if (params.playerTwo > playerOne) {
```

Comment:
Line 36 is so close! You just need to add a 'params.' in front of playerOne so the function knows what playerOne is (in this instance that it needs to grab it from the params we've passed in from the prompt)  ex: 'params.playerOne' ----> ' if(params.playerTwo > params.playerOne)'
Okay, other than that...I think you're saying IF playerTwo's name is longer (and in that way larger) than playerOne's THEN playerOne "can't move like me" right? If so, then you'll want to add '.length' to the end of each so that Javascript knows to count the number of items in each string in order to compare it. ex: 
-----> if('params.playerTwo.length > params.playerOne.length')